src/components/subscriptions/data/hooks.js

```diff
@@ -69,6 +69,7 @@ export const useSubscriptions = ({ enterpriseId, errors, setErrors }) => {
     loadCustomerAgreementData();
   }, [loadCustomerAgreementData]);
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   useEffect(loadCustomerAgreementData, [enterpriseId]);
```

Comment:
We should avoid ignoring ESLint rules about `react-hooks/exhaustive-deps`. Is there a fix here to avoid getting the error?

Reply:
That would be to either add loadLinks to the dependency array or remove the array entirely. I have seen these being ignored for useEffect in other areas/repos as well. One such example would be in `SSOConfigConfiguredCard.jsx`